From 2d68cdf34f13065d4a0d78c95932262c1dc2b7e0 Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Thu, 2 Apr 2015 20:07:03 +0200
Subject: [PATCH] Fix Wayland protocols build on compilers other than gcc

Move the C flags out of the gcc check to actually set them for
every compiler. No need to do any particular check on the compiler
here since this is Wayland stuff and people are going to build it
only on Linux compilers which will likely understand the flags.
---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d23c6a6..c7aef0a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,11 +38,11 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     if ("${CMAKE_CXX_COMPILER_VERSION}" VERSION_LESS "4.8")
         message(SEND_ERROR "Version ${CMAKE_CXX_COMPILER_VERSION} of the ${CMAKE_CXX_COMPILER_ID} C++ compiler is not supported. Please use version 4.8 or later.")
     endif()
-
-    # Adjusting CMAKE_C_FLAGS to get wayland protocols to compile
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu90")
 endif()
 
+# Adjusting CMAKE_C_FLAGS to get Wayland protocols to compile
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu90")
+
 # Add custom uninstall target
 if ("${ECM_VERSION}" VERSION_LESS "1.7.0")
     # ECM 1.7.0 provides this target, so we can't roll our own (see issue #172)
-- 
1.9.0

